tests/test_cases/test_cocotb/test_timing_triggers.py

```diff
@@ -154,19 +157,20 @@ def test_cached_write_in_readonly(dut):
         raise TestFailure
 
 
-@cocotb.test(expect_fail=cocotb.SIM_NAME.lower().startswith(("icarus",
-                                                             "chronologic simulation vcs")),
-             skip=cocotb.SIM_NAME.lower().startswith(("ncsim", "xmsim")))
-def test_afterdelay_in_readonly(dut):
-    """Test doing invalid sim operation"""
-    global exited
-    exited = False
-    clk_gen = cocotb.fork(Clock(dut.clk, 100).start())
-    coro = cocotb.fork(do_test_afterdelay_in_readonly(dut, 0))
-    yield [Join(coro), Timer(1000)]
-    clk_gen.kill()
-    if exited is not True:
-        raise TestFailure
+# Suggestion is to scrap this completely, as we now warn about Timer(0) being problematic
+# @cocotb.test(expect_fail=cocotb.SIM_NAME.lower().startswith(("icarus",
+#                                                              "chronologic simulation vcs")),
+#              skip=cocotb.SIM_NAME.lower().startswith(("ncsim", "xmsim")))
+# def test_afterdelay_in_readonly(dut):
+#     """Test doing invalid sim operation"""
+#     global exited
+#     exited = False
+#     clk_gen = cocotb.fork(Clock(dut.clk, 100).start())
+#     coro = cocotb.fork(do_test_afterdelay_in_readonly(dut, 0))
+#     yield [Join(coro), Timer(1000)]
+#     clk_gen.kill()
+#     if exited is not True:
+#         raise TestFailure
```

Comment:
```suggestion
```

We don't let dead commented out code lie around.

Reply:
yes I just left it like this to await some further comments but to move on to get this fixed finally. we can remove now.

Reply:
accepted it, but also removed to additional empty lines, otherwise py linters will complain